cmd/service/logs.go

```diff
@@ -27,20 +25,18 @@ func init() {
 }
 
 func logsHandler(cmd *cobra.Command, args []string) {
-	serviceID := args[0]
+	go showLogs(args[0], cmd.Flag("dependency").Value.String())
+	<-utils.WaitForCancel()
+}
+
+func showLogs(serviceID string, dependency string) (err error) {
 	service, err := serviceDB.Get(serviceID)
-	if err != nil {
-		return
-	}
-	dependency := cmd.Flag("dependency").Value.String()
+	utils.HandleError(err)
 	readers, err := service.Logs(dependency)
-	if err != nil {
-		fmt.Println(aurora.Red(err))
-		return
-	}
+	utils.HandleError(err)
 	for _, reader := range readers {
 		defer reader.Close()
-		go stdcopy.StdCopy(os.Stdout, os.Stderr, reader)
+		stdcopy.StdCopy(os.Stdout, os.Stderr, reader)
```

Comment:
Remove the goroutine prevents to see the logs from many dependencies..

Reply:
Adding the goroutine, without having the `<-utils.WaitForCancel()` in the same function make the function executes the `defer reader.Close()`.
We could return a function `close` that will call the `reader.Close()`. What do you think?